csatar-plugins/csatar/models/TeamReport.php

```diff
@@ -243,4 +243,18 @@ public function getDistrict() {
     public function getTeam() {
         return $this->team_id ? $this->team : null;
     }
+
+    // method to get scouts without registration form
+    public static function getScoutsWithoutRegistrationForm($scouts): array
+    {
+        foreach ($scouts as $scout) {
+            if(!$scout->registration_form) {
+                $scoutsWithoutRegistrationForm[] = [
+                    'name' => $scout->name,
+                    'ecset_code' => $scout->ecset_code,
+                ];
+            }
+        }
+        return $scoutsWithoutRegistrationForm ?? [];
+    }
```

Comment:
Ebben a fuggvenyben a $scoutsWithoutRegistrationForm-ba bekerulnek az uj elemek, nem irodnak mindig felul? Lehet ok, csak meg en nem hasznaltam igy az array-t.

Reply:
Erre a része gondolsz: `$scoutsWithoutRegistrationForm[] =`?
Ez ugyanazt csinálja mint az array_push, nekem újabban mindig beleköt a php strom az array_push-ba és ezt javasolja helyette. Állítólag gyorsabb: https://stackoverflow.com/questions/559844/which-is-faster-in-php-array-value-or-array-pusharray-value